libs/base/ui/overlays/modal/src/modal.styles.ts

```diff
@@ -16,7 +16,10 @@ export const styles = css`
       calc(100vw - (var(--oryx-container-bleed, 0px) * 2)),
       var(--oryx-modal-width, 100%)
     );
-    max-width: calc(100vw - var(--oryx-modal-bleed, 0px) * 2);
+    max-width: calc(
+      var(--oryx-container-width, --oryx-modal-width) -
```

Comment:
```suggestion
      var(--oryx-container-width, var(--oryx-modal-width)) -
```

Reply:
we might also need to add a hardcoded fallback maybe to not break the calc in case there's nothing